build/script_windows.ps1

```diff
@@ -0,0 +1,6 @@
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+$PSDefaultParameterValues['*:ErrorAction']='Stop'
+
+Invoke-Expression -Command "cmd.exe /c salt-call --local state.show_sls amazon-inspector"
```

Comment:
Please confirm that this actually errexits by testing with a known-failing command...

Reply:
The script will completely stop running and throw the error. It will not continue on to the other commands.

Reply:
Ok thanks. One more favor... Can you introduce a syntax error into the formula itself and confirm that travis fails?

Reply:
Looks like the intentional formula error is being masked - https://travis-ci.com/confusdcodr/amazon-inspector-formula/builds/98816149

Reply:
This option is necessary for salt to pass through failures in state execution to the return code of salt itself: `--retcode-passthrough`

Reply:
Successfully forced travis to fail with intentional error - https://travis-ci.com/confusdcodr/amazon-inspector-formula/builds/98823446

Reply:
And now we are successfully building again - https://travis-ci.com/confusdcodr/amazon-inspector-formula/builds/98824534